features/auth/signup/FeedbackForm.tsx

```diff
@@ -20,6 +22,14 @@ export default function FeedbackForm() {
         authState.flowState!.flowToken,
         form
       );
+      TagManager.dataLayer({
+        dataLayer: {
+          event: "sign_up",
+          signupMethod: "email",
+          userId: getRandomId(),
```

Comment:
Mostly used to identify a notion of a unique user, but since passing the user ID the user used would be PII (although unsure this form has access to that anyway?), I've replaced it with a random ID generated from the browser which is good enough for this.

Reply:
Actually `window.location.href` might contain someone's flow token 😬 it's probably useless by that point as they've fully signed up, but I think still better not to send that. Will change it when I have more time later...

Reply:
Too bad there's no way to leave this undefined or something to indicate the event was produced by an anonymous user